addOns/automation/CHANGELOG.md

```diff
@@ -4,7 +4,8 @@ All notable changes to this add-on will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 ## Unreleased
-
+### Added
+- Save-as button next to the save button for saving plans.
```

Comment:
e.g. `Add Save As button that allows to save the automation plan to a different file (Issue 7178).`